test/VsixTesting.Xunit.Tests/VsixTesting.Xunit.Tests.csproj

```diff
@@ -27,4 +27,13 @@
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\VsixTesting.Invoker\VsixTesting.Invoker.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\\..\src\VsixTesting.Invoker\VsixTesting.Invoker.csproj" ReferenceOutputAssembly="false" CopyVsix="true" ExtensionsDirectory="CopyVsix/True.Relative" />
```

Comment:
❓ If you **Clean** the test project, are the copied VSIX files removed as well?

Reply:
Yes. The **Clean** target will remove any files found in the `FileWrites` item list.

https://github.com/josetr/VsixTesting/blob/83712a3865826de8284a1097c23044da1705f53d/src/VsixTesting.Xunit/VsixTesting.Xunit.targets#L34